var/spack/repos/builtin/packages/gplates/package.py

```diff
@@ -0,0 +1,75 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Gplates(CMakePackage):
+    """GPlates is desktop software for the interactive visualisation of
+    plate-tectonics. GPlates offers a novel combination of interactive
+    plate-tectonic reconstructions, geographic information system (GIS)
+    functionality and raster data visualisation. GPlates enables both the
+    visualisation and the manipulation of plate-tectonic reconstructions
+    and associated data through geological time."""
+
+    homepage = 'https://www.gplates.org'
+
+    version('2.0.0', '9e95874b35a01f4c9bff5845a1621ad5')
+
+    def url_for_version(self, version):
+        return 'https://sourceforge.net/projects/gplates/files/gplates/{0}/gplates-{1}-unixsrc.tar.bz2/download'.format(version.up_to(2), version)
```

Comment:
I usually put `url_for_version` after all directives (depends_on, conflicts, etc) and at the start of all function definitions.

Reply:
Thanks! Apparently, I forgot to move this down after adding all the dependencies. :-)